src/python/pants/base/exception_sink.py

```diff
@@ -114,22 +104,26 @@ class ExceptionSink:
     _interactive_output_stream = None
     # Whether to print a stacktrace in any fatal error message printed to the terminal.
     _should_print_backtrace_to_terminal = True
-    # An instance of `SignalHandler` which is invoked to handle a static set of specific
-    # nonfatal signals (these signal handlers are allowed to make pants exit, but unlike SIGSEGV they
-    # don't need to exit immediately).
```

Comment:
This information was potentially useful... did you determine that it is no longer accurate?

Reply:
Nope that was an extraneous delete, restoring